hypertrace-gradle-docker-java-application-plugin/src/main/java/org/hypertrace/gradle/docker/HypertraceDockerJavaApplication.java

```diff
@@ -26,6 +28,7 @@ public HypertraceDockerJavaApplication(
     this.serviceName = objectFactory.property(String.class)
                                     .convention(projectName);
     this.port = objectFactory.property(Integer.class);
+    this.ports = objectFactory.listProperty(Integer.class);
```

Comment:
Does any of this have preference over the other? What if I set both? Is it worth to clarify in the code or the docs?

Reply:
No preference - they're both added to the exposed port list. The only difference in behavior is how they affect the admin port's default value, which is already included in the docs. Will update docs to clarify the behavior of the two.